app/src/main/java/org/sopt/and/ui/screen/signup/composable/SignUpContentScreen.kt

```diff
@@ -0,0 +1,102 @@
+package org.sopt.and.ui.screen.signup.composable
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.imePadding
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.core.content.ContextCompat
+import org.sopt.and.R
+import org.sopt.and.ui.component.button.FullWidthTextButton
+import org.sopt.and.util.isValidEmail
+import org.sopt.and.util.isValidPassword
+
+@Composable
+fun SignUpScreen(
+    modifier: Modifier = Modifier,
+    onSignUpComplete: (String, String) -> Unit = { _, _ -> }
```

Comment:
아래와 같이 바꿔도 좋을 것 같네요 !!
```suggestion
    onSignUpComplete: (email: String, password: String) -> Unit = { _, _ -> }
```

Reply:
새로 배워갑니다 !! 감사합니다 👍